addons/test_import_export/tests/test_import.py

```diff
@@ -992,14 +998,83 @@ def test_xlsx_datetime_values_assigned_to_wrong_field(self):
                 'datetime_format': options.get('datetime_format'),
             }
         )
-        self.assertTrue(response.get('messages'), "Expected error messages due to date/time values in non-date fields")
-        self.assertEqual(len(response['messages']), 2)
+        self.assertFalse(response.get('messages'))
+        self.assertTrue(all(isinstance(name, str) for name in response['name']))
```

Comment:
I think it should be nice to check the value of `name` in this case.
Even if the result is not perfect, it better to test and explain this corner case.